client/src/pages/TicketCreation.js

```diff
@@ -1,11 +1,202 @@
+      {success && <div className="alert alert-success">{success}</div>}
+
+      <form onSubmit={handleSubmit}>
+        <div className="mb-3">
+          <label htmlFor="id" className="form-label fw-bold">
+            ID <span className="text-danger">*</span>
+          </label>
+          <input
+            type="text"
+            className="form-control "
+            id="id"
+            value={id}
+            onChange={(e) => setId(e.target.value)}
+            required
+          />
+        </div>
+
+        <div className="mb-3">
+          <label htmlFor="description" className="form-label fw-bold">
+            Description <span className="text-danger">*</span>
+          </label>
+          <input
+            type="text"
+            className="form-control"
+            id="description"
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+            required
+          />
+        </div>
+
+        <div className="mb-3">
+          <label htmlFor="status" className="form-label fw-bold">
+            Status <span className="text-danger">*</span>
+          </label>
+          <select
+            className="form-control"
+            id="status"
+            value={status}
+            onChange={(e) => setStatus(e.target.value)}
+            required
+          >
+            <option value="">Select Status</option>
+            <option value="Completed">Completed</option>
+            <option value="Assigned">Assigned</option>
+            <option value="Open">Open</option>
+            <option value="Cancelled">Cancelled</option>
+            <option value="WIP">WIP</option>
+          </select>
+        </div>
+
+        <div className="mb-3">
+          <label htmlFor="location" className="form-label fw-bold">
+            Location <span className="text-danger">*</span>
+          </label>
+          <input
+            type="text"
+            className="form-control"
+            id="location"
+            value={location}
+            onChange={(e) => setLocation(e.target.value)}
+            required
+          />
+        </div>
+
+        <div className="mb-3">
+          <label htmlFor="request_date" className="form-label fw-bold">
+            Request Date <span className="text-danger">*</span>
+          </label>
+          <input
+            type="date"
+            className="form-control"
+            id="request_date"
+            value={request_date}
+            onChange={(e) => setRequestDate(e.target.value)}
+            required
+          />
+        </div>
+
+        <div className="mb-3">
+          <label htmlFor="shop" className="form-label fw-bold">
+            Shop <span className="text-danger">*</span>
+          </label>
+          <input
+            type="text"
+            className="form-control"
+            id="shop"
+            value={shop}
+            onChange={(e) => setShop(e.target.value)}
```

Comment:
There is a fetchShop request in my branch. After we merge it, we can change it to a shop-selection that allow user select instead of taking whatever they input.